libfreerdp/core/settings.c

```diff
@@ -648,7 +648,7 @@ rdpSettings* freerdp_settings_new(DWORD flags)
 	    !freerdp_settings_set_bool(settings, FreeRDP_NSCodecAllowDynamicColorFidelity, TRUE) ||
 	    !freerdp_settings_set_bool(settings, FreeRDP_AutoReconnectionEnabled, FALSE) ||
 	    !freerdp_settings_set_uint32(settings, FreeRDP_AutoReconnectMaxRetries, 20) ||
-	    !freerdp_settings_set_bool(settings, FreeRDP_GfxThinClient, TRUE) ||
+	    !freerdp_settings_set_bool(settings, FreeRDP_GfxThinClient, FALSE) ||
```

Comment:
Shall we document that in the merging to FreeRDP 3.0 doc, as in some cases that can be a pretty important detail (those who have played with egfx proxying know what I'm talking about)

Reply:
@hardening which document would that be? for the client command line that would now fit the doc as that was wrong.

Reply:
Oh correct forget my remark